Automattic/woocommerce-payments
# Pin development PHP dependencies to specific versions

#### Changes proposed in this Pull Request

The dependencies that Renovate is trying to pin in https://github.com/Automattic/woocommerce-payments/pull/2440 don't seem to resolve. However, we I do them manually in my local environment they do.

~~Let's see what the CI pipeline does with this PR....~~ OK, it works. I think this is OK to merge then, although it would be great if anyone can spot what's tripping up the other PR.

#### Testing instructions

The other PR has every CI job failing, so they should pass and we should be able to run `composer install` successfully.

-------------------
